### PR TITLE
[67] Style the Product Details Page using CSS- Frontend

### PR DESCRIPTION
**Pull request by Onyinye101**
## Task: `[67] Style the Product Details Page using CSS`
- `styled the product details page using CSS`

Screenshot 2020-11-06 at 11 14 17 AM
Screenshot 2020-11-06 at 11 14 35 AM
Screenshot 2020-11-06 at 11 15 04 AM
Screenshot 2020-11-06 at 11 15 32 AM

### CODE REVIEW

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 1]
```diff
@@ -1,71 +1,117 @@
...
-                    <li> <a href="#"> Cart </a></li>
-                </ul>
+                <div class="left-side">
+                    <div class="tab">
+                        <strong>BOOTSTRAPS</strong>
+                    </div>
+                </div>
+                <div class="right-side">
+                    <div class="link-wrapper">
+                        <a href="../html/index.html"> Product Page </a>
```
Comment (Ireti203) [inline submissions/onyinyeChukwuka/HTML/index.html:24]: Format this line properly

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 2]
```diff
@@ -1,71 +1,117 @@
...
-            <br>
-            - Heavy laced leather boot especially with a wide cuff buckled above the ankle. Suitable for outdoor activities and casual outings.
-        </p>
+            <!-- FOR THE RIGHT SIDE -->
+            <div class="camidescription">
+                <h3> CAMI'S COMBAT BOOTS</h3>
+                <h4> Brand: River Island</h4>
+                <h4>Quantity: <input type="text" value="1" name="quantity " id="quantity"></h4>
+                <h4> Price: &#x20A6 15,000</h4>
+                <button type="submit" id="button"> BUY NOW</button>
```
Comment (Ireti203) [inline submissions/onyinyeChukwuka/HTML/index.html:47]: And here too.

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 3]
```diff
@@ -1,71 +1,117 @@
...
+            </div>
+        </div>
+
+        <!-- FOR THE DESCRIPTION -->
+        <div class="description">
+            <h4> DESCRIPTION</h4>
```
Comment (Ireti203) [inline submissions/onyinyeChukwuka/HTML/index.html:53]: Format this line properly, check for other lines with unbalanced spaces between tags and text too.

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 4]
```diff
@@ -1,71 +1,117 @@
...
-                <li> <a href="#"> Cami's Platform boots </a></li>
-                <li> <a href="#"> Military-style boots </a></li>
-            </ol>
-        </p>
-    </aside>
-    <footer>
-        <p>© 2020 BootStraps</p>
+    <!-- for the related products section -->
+    <div class="related">
+        <h4> RELATED PRODUCTS</h4>
```
Comment (Ireti203) [inline submissions/onyinyeChukwuka/HTML/index.html:60]: Same here.

File: submissions/onyinyeChukwuka/css/productpage.css [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+.footer {
+    background-color: #0066f5;
+    padding: 20px;
+    color: #fcf8f8;
+}
+
+.facebook{
+    width: 45px;
+    height: 41px;
+}
```
Comment (Ireti203) [inline submissions/onyinyeChukwuka/css/productpage.css:160]: Your formatting from 160-172 is not consistent with what you have on previous lines.

**Review by Ireti203 — COMMENTED** [review]
Weldone @Onyinye101

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 6]
```diff
@@ -1,71 +1,117 @@
...
-                    <li> <a href="../html-forms/contactform.html" target="_blank"> Contact Us </a></li>
-                    <li> <a href="#"> Sign in </a></li>
-                    <li> <a href="#"> Cart </a></li>
-                </ul>
+                <div class="left-side">
+                    <div class="tab">
+                        <strong>BOOTSTRAPS</strong>
+                    </div>
+                </div>
+                <div class="right-side">
```
Comment (TundeOladejo) [inline submissions/onyinyeChukwuka/HTML/index.html:22]: same here too change the class name

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 7]
```diff
@@ -1,71 +1,117 @@
...
-                <li> Chunky sole</li>
-                <li> Lace-up fastening</li>
-            </ul>
-        </p>
-        <p>
-            <strong> DESCRIPTION</strong>
-            <br>
-            - Heavy laced leather boot especially with a wide cuff buckled above the ankle. Suitable for outdoor activities and casual outings.
-        </p>
+            <!-- FOR THE RIGHT SIDE -->
```
Comment (TundeOladejo) [inline submissions/onyinyeChukwuka/HTML/index.html:41]: When another developer is going through your code and they don't have your UI, this comment might be a lil' bit confusing.

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 8]
```diff
@@ -1,71 +1,117 @@
...
-        </p>
-    </aside>
-    <footer>
-        <p>© 2020 BootStraps</p>
+    <!-- for the related products section -->
+    <div class="related">
+        <h4> RELATED PRODUCTS</h4>
+        <div class="items">
+            <div class="martens">
+                <a href="#"> <img src="../html/images/drmartens.png" alt="Dr-martens-boots" width="399px"
```
Comment (TundeOladejo) [inline submissions/onyinyeChukwuka/HTML/index.html:63]: Format properly

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 9]
```diff
@@ -1,71 +1,117 @@
...
+            </div>
+
+            <div class="socials">
+                <div class="newsletter">
+                    <label for="email">Sign up for our Newsletter</label><br><br>
+                    <div class="newsletter-input">
+                        <input type="text" id="newslettter" name="email" placeholder="Enter your email address">
+                        <button type="button" id="submitbutton">SUBMIT</button>
+                    </div>
+
```
Comment (TundeOladejo) [inline submissions/onyinyeChukwuka/HTML/index.html:98]: Remove extra space

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 10]
```diff
@@ -1,71 +1,117 @@
...
+                </div>
+
+                <div class="connect">
+                    <p> CONNECT WITH US</p>
+                    <div class="images">
```
Comment (TundeOladejo) [inline submissions/onyinyeChukwuka/HTML/index.html:103]: Please use a better class name to describe the logos in this footer. `Images` is confusing

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 11]
```diff
@@ -1,71 +1,117 @@
...
+                        height="298px"></a>
+                <h4> Chain Platform Boots</h4>
+                <h4> Price: &#x20A6 35,000</h4>
+            </div>
+            <div class="military">
+                <a href="#"> <img src="../html/images/military.jpg" alt="Dr-martens-boots" width="399px"
+                        height="298px"></a>
+                <h4>Speedlace Military Boots</h4>
+                <h4> Price: &#x20A6 25,500</h4>
+            </div>
```
Comment (TundeOladejo) [inline submissions/onyinyeChukwuka/HTML/index.html:79]: I can see that you have created 3 different classes inside the `items` class div. One class should be enough for the 3 items list that you have. I will advise that you use a class name for the 3 items. A class name like `items-list` will be proper to describe them.

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

File: submissions/onyinyeChukwuka/HTML/index.html [hunk 12]
```diff
@@ -1,71 +1,117 @@
...
+                <h4> Price: &#x20A6 75,000</h4>
+            </div>
+            <div class="platforms">
+                <a href="#"> <img src="../html/images/platform.jpg" alt="Dr-martens-boots" width="399px"
+                        height="298px"></a>
+                <h4> Chain Platform Boots</h4>
+                <h4> Price: &#x20A6 35,000</h4>
+            </div>
+            <div class="military">
+                <a href="#"> <img src="../html/images/military.jpg" alt="Dr-martens-boots" width="399px"
```
Comment (TundeOladejo) [inline submissions/onyinyeChukwuka/HTML/index.html:75]: I notice that you have the same `alt` attribute values for the 3 products which are not cool enough. Use different alternate texts to describe the products.

File: submissions/onyinyeChukwuka/HTML/index.html (hunk 1 above)
Comment (Onyinye101) [reply submissions/onyinyeChukwuka/HTML/index.html:24]: what do you mean @Ireti203

Comment (Ireti203) [reply submissions/onyinyeChukwuka/HTML/index.html:24]: Compare line 24 and 27. The spacing between Product Page and the tags. @Onyinye101

**Review by adaugochi — APPROVED** [review]

**Pull request merged**